partner_medical_medication/models/res_partner.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    medication_ids = fields.Many2many(
+        string='Medications',
+        comodel_name='medical.patient.medication',
+        compute='_compute_medication_ids',
+        store=True,
```

Comment:
The flipside being that the database is updated each time one of these attributes is, even if it isn't being viewed. I think removing the store as @jcdrubay suggested may actually be better in hindsight.